test/run/test_assemblers.py

```diff
@@ -220,11 +221,29 @@ def test_tar(self):
                 with self.run_assembler(osb,
                                         "org.osbuild.tar",
                                         options,
-                                        filename) as (_, image):
+                                        filename) as (tree, image):
                     output = subprocess.check_output(["file", "--mime-type", image], encoding="utf-8")
                     _, mimetype = output.strip().split(": ") # "filename: mimetype"
                     self.assertIn(mimetype, expected_mimetypes)
 
+                    if compression:
+                        continue
```

Comment:
Why don't we verify compressed files?

Reply:
compression just means if the tarball itself is compressed, the content, i.e. the tree in the manifest is exactly the same, so I thought this is just a waste of computational resources.